entrypoint.sh

```diff
@@ -11,6 +11,17 @@ function appendParams(){
 
 kn_command=("kn" "service")
 
+namespace=""
```

Comment:
If this contained just the namespace, this would be a good name for it. But, since it's going to contain "--namespace", I think we should clarify that in the name - like `namespace_option` or `namespace_arg`.

Reply:
Yes, nice suggestion.
Was not able to find any good name at that time.